signalfx-org-metrics/README.md.jinja

```diff
@@ -11,43 +11,72 @@
 
 ## DESCRIPTION
 
-Splunk Infrastructure Monitoring generates a number of metrics you can use to monitor your usage of the application, including:
+Splunk Infrastructure Monitoring provides the following types of metrics that measure your usage:
```

Comment:
how do you feel about shorter bullet points? e.g., dropping "such as, ..." from each bullet point? also, i think if one bullet point requires a period at the end, all of them should have a period at the end.

Reply:
Shorter is better, but rewriting the entire point might be better than just dropping words.

Bullet points must use parallel construction, _but_ when you have to terminate a phrase and add a new sentence, both get a period. Phrases don't get a period.